filesForReview/script.js

```diff
@@ -0,0 +1,84 @@
+const getPokemon = async (pokemonName) => {
+  return fetch(`https://pokeapi.co/api/v2/pokemon/${pokemonName}`)
+  .then((response) => response.json())
+  .catch((error) => error);
+}
+
+const configureChartData = (stats, pokemonName) => {
+  const baseStats = stats.map((stat) => stat.base_stat);
+  const [hp, atk, def, spAtk, spDef, speed] = baseStats;
+  return {
+    labels: ['HP', 'Attack', 'Special Attack', 'Speed', 'Special Defense', 'Defense'],
+    datasets: [{
+      label: `${pokemonName}`,
+      data: [hp, atk, spAtk, speed, spDef, def],
+      fill: true,
+      backgroundColor: 'rgb(3, 4, 94, 0.2)',
+      borderColor: '#03045eff',
+      pointBackgroundColor: '#fff',
+      pointBorderColor: '#03045eff',
+      pointHoverBackgroundColor: '#03045eff',
+      // pointHoverBorderColor: '#fff'
+    }]
+  }
+};
+
+const configureChart = (callback, stats, pokemonName) => {
+  return {
+    type: 'radar',
+    data: callback(stats, pokemonName),
```

Comment:
Mando muito aplicando os conhecimentos de callback!
Mas acho que nessa função não era tão necessário e você poderia simplesmente chamar a função ali mesmo.
```suggestion
const configureChart = (stats, pokemonName) => {
  return {
    type: 'radar',
    data: configureChartData(stats, pokemonName),
```
Isso por que nós geralmente utilizamos callback para que sua função fique mais flexivel, a callback seria útil caso existisse uma função configureChartData diferente, e em dado momento você precisasse passar uma ou outra, sacou?